fe/src/components/Tab/ClientTab.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <v-container>
+   <v-navigation-drawer class="pr-2" permanent position="fixed">
+      <v-list dense>
+        <v-list-item-group>
+          <!--Reservations-->
+          <v-list-item position="fixed" href="#reservations" @click="activeOption = 'r'" :class="{'option-active' : activeOption === 'r'}">
+            <v-list-item-content>
+              <v-list-item-title><b>Your Reservations</b></v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+        </v-list-item-group>
+      </v-list>
+    </v-navigation-drawer>
+  </v-container>
+</template>
+
+<script>
+  export default {
+      data: () => ({
+          activeOption: ' '
```

Comment:
what is the data in `activeOption`? if it's limited to two options, please use boolean otherwise use a defualt option